src/background-plugin.ts

```diff
@@ -546,9 +541,9 @@ async function createWorker(
   names: string[],
   modules: WebAssembly.Module[],
   sharedData: SharedArrayBuffer,
-  onworker: (_w: Worker) => void = (_w: Worker) => { },
+  onworker: (_w: Worker) => void = (_w: Worker) => {},
 ): Promise<Worker> {
-  const worker = new Worker(WORKER_URL);
+  const worker = new Worker(WORKER_URL, opts.nodeWorkerArgs);
```

Comment:
This is the actual worker. We've normalized `nodeWorkerArgs` by this point. (See :apple:)